packages/next/server/config.ts

```diff
@@ -650,10 +649,8 @@ export default async function loadConfig(
     }
   }
 
-  const completeConfig = defaultConfig as NextConfigComplete
-  completeConfig.configFileName = configFileName
```

Comment:
Where is `configFileName` assigned? I think you still need to keep it here

Reply:
https://github.com/vercel/next.js/pull/33064/files#diff-1ce54af4c1a0166a265bebae9ba304b24816ebe9cbb92f90b9707379b0680e97R205

Reply:
@balazsorban44 What about when its `next.config.mjs`?